xbmc/utils/StreamDetails.cpp

```diff
@@ -557,8 +557,8 @@ std::string CStreamDetails::VideoDimsToResolutionDescription(int iWidth, int iHe
   // 1280x720
   else if (iWidth <= 1280 && iHeight <= 720)
     return "720";
-  // 1920x1080
-  else if (iWidth <= 1920 && iHeight <= 1080)
+  // 1920x1080 (sometimes 1088, which is a multiple of 16 - see comment about 544)
```

Comment:
Perhaps that size is computed directly out of avctx->height - like in here: https://github.com/xbmc/xbmc/blob/master/xbmc/cores/dvdplayer/DVDCodecs/Video/VAAPI.cpp#L2989 via: https://github.com/xbmc/xbmc/blob/master/xbmc/cores/dvdplayer/DVDCodecs/Video/VAAPI.cpp#L520

then 1088 is quite clear